DECISION-MAKING.md

```diff
@@ -1,143 +1,154 @@
 `Last update: 2018-07`
 
-# Service owners
+# _Mozilla IAM_ Program Strategy
 
-High level service and product owners for Mozilla IAM.
+* @jeffbryner (IT, InfoSec)
+* @hmitsch (Open Innovation, ParSys)
 
-* Capability owner: @jeffbryner
-  * Product owner: @gdestuynder
+# Mozilla IT Operating Model
+
+* IT Capability Owner: @jeffbryner
+  * IT Product Owner: @gdestuynder
     * Auth0 & Auth0 Lock & Auth0 Dashboard: @jdow @akrug @gdestuynder
+       * <Question: Is `Auth0 Lock` the same as `NLX`>
     * Service Integration (CIS): @akrug
     * LDAP: @jdow
     * DuoSecurity: @gdestuynder
-    * Mozillians.org: @hmitsch
-    * WorkDay: HRIS
-
-## Architects
-
-* @gdestuynder (IAM, Security)
-* @mvk-mozilla (IT)
-
-## Program Managers
-
-* @hmitsch
-
-## User Experience (UX)
-
-* @mbranson
+    
+* IT Enterprise Architecture
+   * @mvk-mozilla
+   
+* Outside-IT Capability Ownership
+    * User Experience (UX): @mbranson
+    * Mozillians.org: ParSys
+    * WorkDay (HRIS): People Team
 
-# Modules owners
+# Modules Owners
 
-This document list which team or persons are responsible for which component (module) of the Mozilla IAM stack. This is
-a more detailed model.
+This document list which team or people are responsible for which component (module) of the _IAM Stack_. This is mainly useful for decision making and is inspired by [Mozilla's Module System](https://wiki.mozilla.org/Modules).
```

Comment:
s/the IAM Stack/Mozilla IAM/

This is the term used everywhere else. Happy to drop "stack" if that's confusing actually.